node/src/test_utils/mod.rs

```diff
@@ -950,19 +992,50 @@ pub mod unshared_test_utils {
                 self
             }
 
-            pub fn permit_to_send_out(mut self) -> Self {
+            pub fn capture_msg_and_let_it_fly_on(mut self) -> Self {
```

Comment:
You'd like to write an explanation for the messages that do not have `Default` trait via stringifying the type name.